src/components/Layout/Header/index.tsx

```diff
@@ -4,68 +4,78 @@ import { Burger } from "./burger"
 import { CreateUser } from "lib//userTag"
 import { LogoutFunc } from "lib/loginFunc"
 import Image from "next/image"
-import icon from "public/monster.png"
+import { logo } from "public"
 import { SearchBar } from "./search"
 
 export const Header: FC = () => {
-
-
   return (
-    <header className="header flex w-full items-center sm:bg-white from-black justify-between px-2 pb6 h-20 z-10">
-      <div className="mobile-view text-white">
+    <header className="header flex w-full items-center from-black justify-between px-2 pb6 h-20 z-10">
+      <div className="mobile-view text-white flex-col">
         <CreateUser />
+        <span className="text-black">
+          {" "}
+          <SearchBar />{" "}
+        </span>
       </div>
       <div className="flex items-center py-4 mx-auto justify-between w-full">
-        <nav className="navbar navbar-expand-lg shadow-md py-2 relative flex items-center w-full">
-          <div className="w-full flex flex-wrap items-center justify-between text-grey">
+        <nav className="navbar navbar-expand-lg shadow-md py-2 relative dark:text-grey flex items-center w-full">
+          <div className="w-full flex flex-wrap items-center justify-between text-grey ">
             <div className="block ml-0 pr-2 lg:px-2 py-2 desktop-view">
-              <Link href="/"
-              ><Image width={10} height={10} className="w-10 h-10 mr-2 rounded-full" src={icon} alt={""}></Image>
+              <Link href="/">
+                <Image
+                  width={70}
+                  height={40}
+                  className="mr-2 overflow-hidden"
+                  src={logo}
+                  alt={""}
```

Comment:
What this alt empty string means 
If image is not loaded there will no be description